### PR TITLE
Edit page header

### PR DESCRIPTION
**Pull request by jlabowitz**
Made the tab in the header for the page you are on be highlighted.

### CODE REVIEW

**Review by tiffperumpail — APPROVED** [review]
LGTM as far as the changes in this PR.

A great addition would be, if either you have time, to align the "Admin" text in the header with the rest of the text.

File: app/assets/stylesheets/discovery.scss [hunk 1]
```diff
@@ -113,4 +113,10 @@
 .navbar-toggle {
   margin-top: 23px;
   padding: 9px 10px !important;
+}
+
+.navbar-text {
+  padding-right: 0.5rem;
+  padding-left: 0.5rem;
+  color: #f00 !important;
```
Comment (ErikOstien) [inline app/assets/stylesheets/discovery.scss:121]: This fixes the spacing on the navbar

File: app/views/application/_header.html.haml [hunk 2]
```diff
@@ -9,20 +9,19 @@
...
               %a{:class => "nav-link", :href => admin_tags_path} Tags
           -else
-            %li{:class => "nav-item"}
+            %li{:class => "nav-item#{' active' if currentPage == 'discovery'}"}
               %a{:class => "nav-link", :href => discovery_path} Discovery
-            %li{:class => "nav-item"}
+            %li{:class => "nav-item#{' active' if currentPage == 'vendor'}"}
               %a{:class => "nav-link", :href => vendors_path} Vendors
-            %li{:class => "nav-item"}
+            %li{:class => "nav-item#{' active' if currentPage == 'product'}"}
```
Comment (ErikOstien) [inline app/views/application/_header.html.haml:24]: Highlighting the tabs to indicate different pages LGTM

File: app/views/vendors/show.html.haml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-= render partial: 'header', locals: {admin: false}
+= render partial: 'header', locals: {currentPage: "vendor", admin: false}
 #details
   #detailsPageHeader
     #detailsPageInfo
```
Comment (ErikOstien) [inline app/views/vendors/show.html.haml:4]: Edit all render partials to be able to highlight current tab LGTM

**Review by ErikOstien — APPROVED** [review]
Navbar is now responsive to which page you are on, and the ADMIN spacing and new red color looks like good work

**Pull request merged**